src/activities/quizzes/QuizEntity.js

```diff
@@ -182,6 +182,26 @@ export class QuizEntity extends Entity {
 		return entity && entity.hasClass(Classes.quizzes.checked);
 	}
 
+	/**
+	 * @returns {bool} Whether or not the user can edit the Deduction Percentage property
+	 */
+	canEditDeductionPercentage() {
```

Comment:
Where is this function used? I only see it used in a test.

Reply:
updated: method now used on line 625